packages/codegen-ui-react/lib/utils/forms/array-field-component.ts

```diff
@@ -1031,7 +1030,7 @@ export const renderArrayFieldComponent = (fieldName: string, fieldLabel: string,
         ),
         factory.createJsxAttribute(
           factory.createIdentifier('inputFieldRef'),
-          factory.createJsxExpression(undefined, factory.createIdentifier(`${lowerCaseFirst(fieldName)}Ref`)),
+          factory.createJsxExpression(undefined, factory.createIdentifier(`${fieldName}Ref`)),
```

Comment:
Resolves undefined error. The ref being referenced here is not declared using lowerCaseFirst.

https://github.com/aws-amplify/amplify-codegen-ui/blob/main/packages/codegen-ui-react/lib/forms/react-form-renderer.ts#L435